Test out teams / round robin review for backend dependabot PRs

emmastephenson opened the pull request:
# DEVOPS PULL REQUEST

## Related Issue

- Instead of assigning the entire list as reviewers on every Dependabot Gradle PR, assigning the team **should** force GitHub to select two members of the team for each review.

## Changes Proposed

- Future dependabot PRs for Gradle should only have two assigned reviewers

## Additional Information

Current settings for this team:
![Screen Shot 2022-10-14 at 11 42 25 AM](https://user-images.githubusercontent.com/80282552/195918339-2c1c14ce-e131-40ee-b88e-ee14c2649637.png)

## Testing

- I'm going to attempt to assign the team as reviewer on this PR and we'll see if it works!

emmastephenson commented:
@CDCgov/simplereport-backendreviews testing on this PR shows that the round robin seems to work!

One note for the future - the team has to be created AND separately invited to this repository before you can tag them in requests.

alismx reviewed (commented):
I'm excited to get this in! I'm not the most knowledgeable reviewer for Gradle and frontend dependencies but I'd like to be assigned to the PRs so I can be notified when they're merged/deploying. Can I be assigned to everything in some manner and not included in the round-robin?

DanielSass reviewed (approved):
🥳

emmastephenson commented:
> I'm excited to get this in! I'm not the most knowledgeable reviewer for Gradle and frontend dependencies but I'd like to be assigned to the PRs so I can be notified when they're merged/deploying. Can I be assigned to everything in some manner and not included in the round-robin?

@alismx absolutely! I changed the config to include you as a reviewer alongside the BackendReviews team, and updated the team config to exclude you from the round robin (so you wouldn't be assigned "twice"). I did the same thing for Rin, since I assume she's in the same boat.
![Screen Shot 2022-10-14 at 11 59 59 AM](https://user-images.githubusercontent.com/80282552/195921659-91f3dbcf-bcdb-4eec-b232-8f2cf71a697e.png)

zdeveloper reviewed (approved):
LGTM! this is neat!

zdeveloper commented:
I am concerned about the "**Child team members**" we should be asking adults only to review code changes

alismx reviewed (approved):
Looks good to me! :rocket:

mehansen reviewed (approved):
🎉
